### PR TITLE
[MIG] account_invoice_report_grouped_by_picking_sale_mrp to 13.0

### PR DESCRIPTION
**Pull request by chienandalu**
In this PR:

- We recover the hook method used by this module that was left behind in the `account_invoice_report_grouped_by_picking` migration
- Standar migration adapting tests

Pending from
- [x] #191 

cc @Tecnativa TT26913

please review @ernestotejeda @sergio-teruel

### CODE REVIEW

File: account_invoice_report_grouped_by_picking_sale_mrp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Invoice Grouped by Picking - Sale MRP",
+    "summary": "Take into account BoM kits in invoice report " "grouped by picking",
+    "version": "13.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/account-invoice-reporting",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline account_invoice_report_grouped_by_picking_sale_mrp/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

File: account_invoice_report_grouped_by_picking_sale_mrp/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#. module: account_invoice_report_grouped_by_picking_sale_mrp
+#: model:ir.model,name:account_invoice_report_grouped_by_picking_sale_mrp.model_account_invoice
+msgid "Invoice"
+msgstr "Factura"
+
+#~ msgid "Bill of Material"
+#~ msgstr "Lista de materiales"
+
+#~ msgid "Components Total Qty"
+#~ msgstr "Cantidad total de componentes"
```
Comment (victoralmau) [inline account_invoice_report_grouped_by_picking_sale_mrp/i18n/es.po]:
```suggestion
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Code review OK

**Comment by chienandalu** [comment]
Thanks. Anyway, this should wait for this to be approved: https://github.com/OCA/account-invoice-reporting/pull/191 (although I don't understand very well the fix... :sweat_smile: )

**Review by victoralmau — APPROVED** [review]

**Comment by chienandalu** [comment]
Ready to re-review

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**